src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Negotiate/NegotiateStream_Http_Tests.cs

```diff
@@ -0,0 +1,301 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Infrastructure.Common;
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.ServiceModel.Security;
+using Xunit;
+
+public static class NegotiateStream_Http_Tests
+{
+    // The tests are as follows:
+    //
+    // NegotiateStream_*_AmbientCredentials
+    //     Windows: This should pass by default without any code changes
+    //       Linux: This should not pass by default 
+    //              Run 'kinit user@DC.DOMAIN.COM' before running this test to use ambient credentials
+    //              ('DC.DOMAIN.COM' must be in capital letters) 
```

Comment:
I'm generally leery of doing this as it has effect across the entire user session (we can't isolate this to happen only during a run)

If we know the password, we should be able to automate kinit, and if we want tests to run one after another, then this may force our hand to kdestroy after each test happens. We may have to force such tests to run in sequence.